pommerman/agents/player_agent_blocking.py

```diff
@@ -0,0 +1,37 @@
+from time import time
+import click
+
+from . import BaseAgent
+from .. import characters
+from .. import constants
+
+
+class PlayerAgentBlocking(BaseAgent):
+    """Block for keyboard input."""
+
+    def __init__(self, character=characters.Bomber, agent_control='arrows'):
+        super(PlayerAgentBlocking, self).__init__(character)
+        self._character.agent_image = 'agent-player'
```

Comment:
Why are you setting this?

Reply:
@cinjon oops its from a branch I made with customizable icons.  It should not be in this commit, can remove.

Reply:
I removed